packages/go/ein/ad.go

```diff
@@ -35,9 +35,14 @@ func ConvertSessionObject(session Session) IngestibleSession {
 }
 
 func ConvertObjectToNode(item IngestBase, itemType graph.Kind) IngestibleNode {
+	itemProps := item.Properties
+	if itemProps == nil {
+		itemProps = make(map[string]any)
+	}
+
 	return IngestibleNode{
 		ObjectID:    item.ObjectIdentifier,
-		PropertyMap: item.Properties,
+		PropertyMap: itemProps,
```

Comment:
Do we want to support ingest of nodes with null "Properties"?  Or should we instead raise an error?

Reply:
Its fine as is, this lets us support third party ingest a bit easier and loosens up our requirements